src/ciflow-bot.ts

```diff
@@ -275,8 +291,11 @@ export class CIFlowBot {
       ?.map(label => label.name);
     this.owner = this.ctx.payload?.repository?.owner?.login;
     this.repo = this.ctx.payload?.repository?.name;
-    if (!CIFlowBot.allowed_repos.includes(`${this.owner}/${this.repo}`)) {
-      return false;
+    if (this.tracker) {
```

Comment:
I think this check can be moved to the end of the setContext function, usually the comments are not "valid" as they might not have the "@pytorchbot" keyword in it, so we only need to check `this.tracker.loadIssue` if they are valid comments.

Reply:
Actually, this check is redundant, as it's no longer necessary to blacklist ciflow bot on the repo level, as empty issue or missing `pytorch-probot.yml` file would mean it is not enabled for a single user.